pkg/ddc/thin/transform_config.go

```diff
@@ -18,14 +18,30 @@ package thin
 
 import (
 	"encoding/json"
+	"fmt"
+	"strings"
 
 	datav1alpha1 "github.com/fluid-cloudnative/fluid/api/v1alpha1"
+	"github.com/fluid-cloudnative/fluid/pkg/common"
+	"github.com/fluid-cloudnative/fluid/pkg/utils/kubeclient"
+	corev1 "k8s.io/api/core/v1"
 )
 
 func (t *ThinEngine) transformConfig(runtime *datav1alpha1.ThinRuntime,
 	dataset *datav1alpha1.Dataset, targetPath string) (config Config, err error) {
 	mounts := []datav1alpha1.Mount{}
+	pvAttributes := map[string]*corev1.CSIPersistentVolumeSource{}
 	for _, m := range dataset.Spec.Mounts {
+		if strings.HasPrefix(m.MountPoint, common.VolumeScheme.String()) {
+			pvcName := strings.TrimPrefix(m.MountPoint, common.VolumeScheme.String())
+			csiInfo, err := t.extractCSIVolumeSourceInfo(pvcName)
+			if err != nil {
+				return config, err
+			}
+
+			pvAttributes[pvcName] = csiInfo
```

Comment:
What is this parameter used for? Only see that this parameter is parsed, but not used.

Reply:
I seem to understand a bit, just update the PV info into a config file that has been mounted in thin fuse pod?